testflinger_agent/client.py

```diff
@@ -267,10 +292,42 @@ def post_agent_data(self, data):
         :param data:
             dict of various agent data points to send to the api server
         """
-        agent = self.config.get("agent_id")
         agent_data_uri = urljoin(self.server, "/v1/agents/data/")
-        agent_data_url = urljoin(agent_data_uri, agent)
+        agent_data_url = urljoin(agent_data_uri, self.config.get("agent_id"))
         try:
             self.session.post(agent_data_url, json=data, timeout=30)
         except RequestException as exc:
             logger.error(exc)
+
+    def post_influx(self, job_id, phase, duration, result):
+        """Post the relevant data points to testflinger server
+
+        :param data:
+            dict of various agent data points to send to the api server
+        """
```

Comment:
There's no need to do anything else below this line if self.influx_client doesn't exist.
```python
if not self.influx_client:
    return
```

Reply:
Yeah, this works as well. Might as well skip defining data.